Core/src/IfcGeometryConverter/CurveConverter.h

```diff
@@ -1001,73 +1001,75 @@ namespace OpenInfraPlatform {
 					//	END_ENTITY;
 					// **************************************************************************************************************************
 					// determine position
-					carve::math::Matrix conic_position_matrix = placementConverter->convertIfcAxis2Placement(circle->Position);
+					carve::math::Matrix conicPositionMatrix = placementConverter->convertIfcAxis2Placement(circle->Position);
 
 					// Get radius
-					double circle_radius = 0.0;
-					if (circle->Radius) {
-						circle_radius = circle->Radius * UnitConvert()->getLengthInMeterFactor();
-					}
-					else {
-						throw oip::InconsistentGeometryException(circle, "No radius!");
-					}
-
-					carve::geom::vector<3> circle_center =
-						conic_position_matrix * carve::geom::VECTOR(0, 0, 0);
+					double circleRadius = circle->Radius * UnitConvert()->getLengthInMeterFactor();
+					
+					carve::geom::vector<3> circleCenter =
+						conicPositionMatrix * carve::geom::VECTOR(0, 0, 0);

```

Comment:
Why is this even needed in the function - it is never used?

Reply:
It is used by function getAngleOnCircle (line 1017 and  line 1024)

Reply:
Resolved: 7826079